Sources/PitchSpeller/Data Structures/Pairable.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Pairable.swift
+//  PitchSpeller
+//
+//  Created by Benjamin Wetherfield on 6/30/18.
+//
+
+protocol Pairable {
+    associatedtype A
+    associatedtype B
+    var a: A { get }
+    var b: B { get }
+    var tuple: (A,B) { get }
```

Comment:
Is this necessary? I am thinking this could be up to the conforming type how it wants to implement this. Looking down at `Swappable`, I feel that `.swapped` is the only necessary extension made (which could be implemented as `var swapped: Self { return .init(a,b) }`)